packages/abi-gen-wrappers/README.md

```diff
@@ -71,3 +71,8 @@ yarn lint
 ```bash
 yarn test
 ```
+
+### Reading Documentation
+
+Documentation for this package is generated by TypeDoc, using the Solidity source code for 0x contracts. Each contract corresponds to one global-level module, which contains relevant enums and interfaces for its events and structs. Most significantly, each module exports a class, `<ContractName>Contract`, e.g. `ExchangeContract`, which implements helper methods for all the functions defined in the corresponding contract.
+A convention to note is that these contract-specific helper methods are defined as _object literals_, which are separated from methods in the generated documentation. Each contract method has a number of sub-methods, e.g. `sendTransactionAsync`, or `estimateGasAsync`, which are documented separately. This is an example of an expected method call signature: `exchangeContractInstance.fillOrder.sendTransactionAsync(...arguments)`.
```

Comment:
If you used a newline here to denote a new paragraph, it's not working.  (Github's "rich diff" view is showing that this line and the previous one are rendering as one big paragraph.)